src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/utils.ts

```diff
@@ -7,17 +8,35 @@ import { AktivitetType, DelvilkårAktivitet } from '../typer/aktivitet';
 import { SvarJaNei } from '../typer/vilkårperiode';
 import { BegrunnelseGrunner } from '../Vilkårperioder/EndreVilkårperiode/utils';
 
-export const nyAktivitet = (behandlingId: string): EndreAktivitetForm => {
-    return {
-        behandlingId: behandlingId,
-        type: '',
-        fom: '',
-        tom: '',
-        aktivitetsdager: undefined,
-        delvilkår: { '@type': 'AKTIVITET' },
-    };
+export const nyAktivitet = (
+    behandlingId: string,
+    aktivitetFraRegister: Registeraktivitet | undefined
+): EndreAktivitetForm => {
+    return aktivitetFraRegister
+        ? {
```

Comment:
Kan ev trekke ut dette til en egen metode som blir brukt her for å få enklere elvis

Reply:
Enig! To ulike funksjoner for "tom" og "tom basert på register"

Reply:
👍